autoload/javacomplete/classpath/gradle.vim

```diff
@@ -56,8 +56,8 @@ function! s:GenerateClassPath(path, gradle) abort
   let s:temporaryGradleFile = tempname()
   let s:gradleOutput = []
   let s:gradlePath = a:path
-  let gradle = has("win32") || has("win16") ? '.\gradlew.bat' : './gradlew'
-  if !executable(gradle)
+  let gradle = javacomplete#util#IsWindows() ? '.\gradlew.bat' : './gradlew'
+  if g:JavaComplete_EnableGradle && executable('gradle')
```

Comment:
ok I know what you me,here I just provid a g:var to switch between `gradle` and `./gradlew`,not to disable gradle featue, may be we should rename this var